### PR TITLE
Reporting Feature

### PR DESCRIPTION
**Pull request by aSquare14**
@julianguyen  Sorry for the mess. Reopened the PR. Please review it whenever free.
Based on issue #14

### CODE REVIEW

File: app/views/reports/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<% title "#{'New Report'}" %>
+<%= form_for(:search, :url => {:action => "create", :ally_id => params[:ally_id], :comment_id => params[:comment_id]}, :method => "post") do |f| %>   
+    <div class="table">
```
Comment (julianguyen) [inline app/views/reports/new.html.erb]: I recently got rid of the table and field classes. So you don't need them and also the div with the clear class. If you need spacing between blocks, use `smallMarginBottom` or `smallMarginTop` depending on your needs.

File: config/locales/en.yml [hunk 2]
```diff
@@ -64,6 +64,7 @@ en:
     add_all: 'Add all'
     select_all: 'Select all'
     remove: 'Remove'
+    report: 'Report'
```
Comment (julianguyen) [inline config/locales/en.yml]: All of these new keys will need be translated to our other languages.

**Comment by julianguyen** [comment]
Hey @prateksha and @aSquare14, what's your status on finishing this up? :) Let me know if any help is needed with resolving conflicts.

**Comment by aSquare14** [comment]
We will finish it by next week ! :) Yes, will let you know. Tysm :heart:

**Comment by julianguyen** [comment]
Just checking in - any updates on this?

**Comment by prateksha** [comment]
We will do it in the next 4-5 days. We got a little busy with our University work. Extremely sorry for the delay.

**Comment by aSquare14** [comment]
@julianguyen Ready for review

**Comment by julianguyen** [comment]
Thanks for updating things! Looks like there's still a bunch of merge conflicts. I can help out if school is busy (which I totally understand).

**Comment by julianguyen** [comment]
Merging this into the `report-feature` branch, will clean up things and post a new PR :)

**Pull request closed without merging**